### PR TITLE
[PTBER072022] berndHavekost

### PR DESCRIPTION
**Pull request by hvkst**

### CODE REVIEW

File: src/chronometer-centiseconds.js [hunk 1]
```diff
@@ -1,37 +1,51 @@
 class Chronometer {
-  constructor() {
-    // ... your code goes here
-  }
+  currentTime = 0;
```
Comment (trebeljahr) [inline src/chronometer-centiseconds.js:2]: haha nice :)

**Review by trebeljahr — COMMENTED** [review]
as always. Good job :) 

I like the little comments, are those things you found out from the others after you looked over their solutions?

File: src/chronometer-centiseconds.js (hunk 1 above)
Comment (trebeljahr) [reply src/chronometer-centiseconds.js:2]: who needs constructors anyways!

Comment (hvkst) [reply src/chronometer-centiseconds.js:2]: Oh, yes... it passed, so I didn´t notice

**Pull request closed without merging**